### PR TITLE
Add React Context and Change WithDataFetch to hook call

### PR DESCRIPTION
**Pull request by Maastrich**

### CODE REVIEW

File: renderer/pages/_app.tsx [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
-  }, [])
+function AppContent(props: AppProps) {
+  const { currentTheme } = useContext(GlobalContext)
+  const { Component, pageProps } = props;
+  return (
+    <ThemeProvider theme={ currentTheme === 'light' ? themeLight : themeDark }>
+      <SnackbarProvider autoHideDuration={ 5000 }>
+        <CssBaseline/>
+        <Skeleton>
+          <Component { ...pageProps } />
```
Comment (Maastrich) [inline renderer/pages/_app.tsx:21]:
```suggestion
          <Component { ...pageProps }/>
```

File: renderer/hooks/useStandardizer.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import ArchiveManager from '@modules/ArchiveManager'
+import { GlobalContext } from 'renderer/context/Store'
+import { capitalize } from '@material-ui/core'
+import { GetterData } from '@d4data/archive-lib/dist/src/types/standardizer/GetterReturn'
+
+interface UseStandardizerArgs {
+  componentName?: string,
+  Component?: FunctionComponent<{ data: NonNullable<GetterData<any>> }>
+  getterArgs?: Array<any>
+}
```
Comment (super-bunny) [inline renderer/hooks/useStandardizer.tsx:13]: Why these arguments ?

File: renderer/pages/dashboard/index.tsx [hunk 3]
```diff
@@ -50,46 +54,64 @@ const Dashboard = () => {
...
+              />
+            </Grid>
+          )}
+        </Grid>
+        <br/>
+        <div>
+          {archiveMetadata?.service && generateTeamNotes()}
+        </div>
+      </Container>
+    )
```
Comment (super-bunny) [inline renderer/pages/dashboard/index.tsx:98]: Extract this to a dedicated component (`Dashboard`) would be cleaner.

File: renderer/pages/dashboard/index.tsx [hunk 4]
```diff
@@ -50,46 +54,64 @@ const Dashboard = () => {
...
+  }
+
+  if (data.data === null) {
+    return (
+      <NoDataAvailable componentName={ capitalize(componentName) }/>
+    )
+  }
+
+  return (
+    Component ? <Component data={ data.data }/> : null
```
Comment (super-bunny) [inline renderer/pages/dashboard/index.tsx:114]: `null` will render a blank page, maybe it's better to display a error message instead ?
Or refactor the component system and display the raw data component here :thinking:

**Review by super-bunny — CHANGES_REQUESTED** [review]
Try to adapt your coding style to the project to avoid minor spacing changes in all files.

File: renderer/hooks/useStandardizer.tsx (hunk 2 above)
Comment (Maastrich) [reply renderer/hooks/useStandardizer.tsx:13]: For options, chatID or rawData's file path

**Review by super-bunny — APPROVED** [review]

**Pull request merged**